### PR TITLE
zh-translation: docs/root/configuration/listeners/network_filters/red…

### PR DESCRIPTION
**Pull request by vgbhfive**
…is_proxy_filter.rst

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #260
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst [hunk 1]
```diff
@@ -1,103 +1,89 @@
...
 
-If a delay is injected, the delay is additive - if the request took 400ms and a delay of 100ms
-is injected, then the total request latency is 500ms. Also, due to implementation of the redis protocol,
-a delayed request will delay everything that comes in after it, due to the proxy's need to respect the 
-order of commands it receives.
+如果注入一个延迟，那么延迟是会累加的。如果请求花费 400ms 再注入 100ms 的延迟，那么总的请求延迟就是 500ms。此外，由于 Redis 协议的实施，因为代理需要维护它所收到的命令顺序，所以延迟请求将会延迟在他之后的所有请求。
 
-Note that faults must have a `fault_enabled` field, and are not enabled by default (if no default value
-or runtime key are set).
+注意，故障应该有一个 `fault_enabled` 字段，并且默认情况下是不启用的（如果没有默认值或在运行时设置值）。
```
Comment (majinghe) [inline docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst]:
```suggestion
注意，故障必须有一个 `fault_enabled` 字段，并且默认情况下是不启用的（如果没有默认值或在运行时设置值）。
```
感觉必须更恰当

**Comment by majinghe** [comment]
@vgbhfive 非常棒的翻译，就一个小 comment 仅供参考。

File: docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst [hunk 2]
```diff
@@ -1,103 +1,89 @@
...
-  downstream_rq_active, Gauge, Total active requests
-  downstream_rq_total, Counter, Total requests
+  downstream_cx_active, Gauge, 活跃连接总数
+  downstream_cx_protocol_error, Counter, 协议异常数
+  downstream_cx_rx_bytes_buffered, Gauge, 当前缓冲中接收的字节数
+  downstream_cx_rx_bytes_total, Counter, 接收的字节数
+  downstream_cx_total, Counter, 总连接数
+  downstream_cx_tx_bytes_buffered, Gauge, 当前缓冲中发送的字节
+  downstream_cx_tx_bytes_total, Counter, 发送的字节数
+  downstream_cx_drain_close, Counter, 由于耗尽而关闭的连接数
```
Comment (addozhang) [inline docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst]:
```suggestion
  downstream_cx_drain_close, Counter, 由于排空关闭的连接数
```

drain 统一翻译成排空

File: docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst [hunk 3]
```diff
@@ -1,103 +1,89 @@
...
 
 .. _config_network_filters_redis_proxy_fault_injection:
 
-Fault Injection
+故障注入
 ---------------
 
-The Redis filter can perform fault injection. Currently, Delay and Error faults are supported.
-Delay faults delay a request, and Error faults respond with an error. Moreover, errors can be delayed.
+Redis 过滤器可以执行故障注入。当然，延迟和异常故障都是支持的。延迟故障延迟请求，异常故障响应异常。此外，异常也可以延迟。
```
Comment (addozhang) [inline docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst]:
```suggestion
Redis 过滤器可以执行故障注入。当前，延迟和异常故障都是支持的。延迟故障延迟请求，异常故障以异常响应。此外，异常也可以延迟。
```

我觉得“响应异常”和“以异常响应”是两种完全不同的意思。

**Review by addozhang — COMMENTED** [review]
提了点建议，麻烦看一下

**Review by majinghe — APPROVED** [review]

**Comment by majinghe** [comment]
@addozhang 晓辉，看一下这个 PR 是否还有问题。

**Review by addozhang — APPROVED** [review]
/LGTM

**Pull request merged**